tests/implementor/test_impl_train_speed_fault.py

```diff
@@ -0,0 +1,62 @@
+from src import implementor as impl
+from src.fault_injector.fault_configurations.train_speed_fault_configuration import (
+    TrainSpeedFaultConfiguration,
+    TrainSpeedFaultConfigurationXSimulationConfiguration,
+)
+
+
+class TestRunImplementor:
```

Comment:
Maybe I missed it in my PR too, but this class should have the name `TestTrainSpeedFaultConfiguration` I think. If so, could you please fix this typo in my old file too please 😇